.github/workflows/main.yml

```
@@ -317,11 +317,11 @@ jobs:
     if: needs.check-changes.outputs.changes-core-client == 'true'
     uses: ./.github/workflows/common-testing-big-instance.yml
     with:
-      working-directory: "./core-client"
-      package-name: "kms-core-client"
+      working-directory: './core-client'
+      package-name: 'kms-core-client'
       # Explicitly skip integration tests since they are all named something with centralized or threshold or full_gen_tests
-      args-tests: "-- --skip centralized --skip threshold --skip full_gen_tests --skip nightly"
-      app-cache-dir: "kms-core-client"
+      args-tests: '-- --skip centralized --skip threshold --skip full_gen_tests --skip nightly --skip centralzd --skip threshld'
```

not sure why this was added -  I guess we can remove

```suggestion
      args-tests: '-- --skip centralized --skip threshold --skip full_gen_tests --skip nightly'
```